src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: 0 48px 46px 0;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__logo {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    object-fit: cover;
+    max-width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: $medium-text;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $primary-grey;
+    margin-bottom: 16px;
+  }
+
+  &__response {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $primary-grey;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: $bold-text;
+      line-height: 18px;
+    }
+  }
```

Comment:
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 16px;
  }

  &__price-text {
    color: $primary-grey;
  }

  &__price-value {
    font-size: 16px;
    font-weight: $bold-text;
    line-height: 18px;
  }
```

Reply:
I wrote about it last time

Reply:
@Yuriy-Komolov it's okay 🙂  Next time if some of comments is not clear, feel free to tag me in personal channel, I'll explain